static/js/components/Group.jsx

```diff
@@ -199,17 +199,20 @@ const Group = () => {
     );
   }
 
-  // currentUser may not have the "Group admin" role, but can still be the group admin?
-  const currentGroupUser = group?.users?.filter(
-    (group_user) => group_user.username === currentUser.username
-  )[0];
-
-  const isAdmin = (aUser, aGroup) =>
-    aUser &&
-    aGroup.group_users &&
-    aGroup.group_users.filter(
-      (group_user) => group_user.user_id === aUser.id
-    )[0].admin;
+  const isAdmin = (aUser) => {
+    const currentGroupUser = group?.users?.filter(
+      (group_user) => group_user.username === aUser.username
+    )[0];
```

Comment:
Could there ever be a situation when this is length=0?

Reply:
@dmitryduev hmm, I'll have the check this for the case of a non-group-member System admin, will update if necessary. thanks for pointing this out

Reply:
@dmitryduev I just checked and this does not error if the list is length 0, it just returns `undefined`, which is perfect for this check

Reply:
Can also use `shift()` to get first element or null.

Reply:
Works for me.